Index: main.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>from fastapi import FastAPI\r\n\r\napp = FastAPI\r\na = 224
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/main.py b/main.py
--- a/main.py	(revision c4b0b0b908a50e3611cb2d02a2816974ef3fdb15)
+++ b/main.py	(date 1712469073518)
@@ -1,4 +1,4 @@
 from fastapi import FastAPI
 
 app = FastAPI
-a = 224
\ No newline at end of file
+a = 2244
\ No newline at end of file
